napari/_qt/qt_layerlist.py

```diff
@@ -511,7 +511,8 @@ def __init__(self, layer):
         textbox = QLineEdit(self)
         textbox.setText(layer.name)
         textbox.home(False)
-        textbox.setToolTip('Layer name')
+        # Chris Wood changed from 'Layer Name' 2020/05/22
```

Comment:
```suggestion
```
can remove... this is appropriately captured in the git commit history

Reply:
Ok, do I need to remove those and then re-commit?

Reply:
you can also just click the "commit suggestion" button

Reply:
(alternatively, you can delete them in your local branch, commit, then push... it will update here)